content/hardware/02.hero/boards/uno-rev3-smd/features.md

```diff
@@ -8,14 +8,6 @@ The "SMD" stands for surface-mount device, and the microcontroller (ATmega328p)
 
 <FeatureList>
 
-<Feature title="ATmega328P" image="core">
-
-The classic high-performance, low-power AVR® microcontroller. 
-
-<FeatureLink title="Datasheet" url="https://content.arduino.cc/assets/Atmel-7810-Automotive-Microcontrollers-ATmega328P_Datasheet.pdf" download blank/>
-
-</Feature>
-
 <Feature title="EEPROM" image="mcu">
```

Comment:
Can you add a link here to EEPROM? 
- https://docs.arduino.cc/learn/programming/eeprom-guide